### PR TITLE
Added logging for remote storage adapter

### PR DESCRIPTION
**Pull request by gdmello**
Add logging on startup and on any error condition during /read or /write.
Ran into a `Decode Error` during `/write` which was not evident, until logging was added.

### CODE REVIEW

**Comment by gdmello** [comment]
@juliusv / @brancz - please review.

File: documentation/examples/remote_storage/remote_storage_adapter/main.go [hunk 1]
```diff
@@ -179,25 +179,29 @@ func buildClients(cfg *config) ([]writer, []reader) {
 		writers = append(writers, c)
 		readers = append(readers, c)
 	}
+	log.Info("Starting up ... ")
```
Comment (juliusv) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: Could you remove the space before the "..."?

File: documentation/examples/remote_storage/remote_storage_adapter/main.go [hunk 2]
```diff
@@ -179,25 +179,29 @@ func buildClients(cfg *config) ([]writer, []reader) {
...
 	return writers, readers
 }
 
 func serve(addr string, writers []writer, readers []reader) error {
 	http.HandleFunc("/write", func(w http.ResponseWriter, r *http.Request) {
 		compressed, err := ioutil.ReadAll(r.Body)
 		if err != nil {
+			log.Errorln("Read Error: ", err)
```
Comment (juliusv) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: "Error" -> "error" here and in the instances below.

Comment (juliusv) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: `log.Errorln()` automatically adds spaces between string arguments, so you won't need the extra space after the colon.

Comment (gdmello) [inline documentation/examples/remote_storage/remote_storage_adapter/main.go]: ok - all changes done.

**Comment by gdmello** [comment]
@juliusv - feedback addressed.

**Comment by juliusv** [comment]
@gdmello Thanks!

**Pull request merged**